packages/ffe-buttons-react/src/ActionButton.js

```diff
@@ -45,4 +45,6 @@ ActionButton.propTypes = {
     leftIcon: node,
     /** Icon shown to the right of the label */
     rightIcon: node,
+    /** Dark variant */
+    dark: bool,
```

Comment:
In todays designforum meeting we agreed to keep the variabel dark at this time. But we are aware this will result in a breaking change if we decide to add multiple theme support in the future.